recipe/meta.yaml

```diff
@@ -94,7 +94,7 @@ outputs:
         - pyarrow
         - python
         - scipy
-        - anndata >=0.10.1
+        - anndata >=0.12
```

Comment:
anndata dropped support for Python 3.9 upstream while ago in https://github.com/scverse/anndata/commit/dd851287bc996b674f0f35b8f15d33fe2c479b6b (https://github.com/scverse/anndata/pull/1712).

Note that they are only dropping older python versions to follow spec-0 as other scientific python packages are doing (https://github.com/ome/ome-zarr-py/issues/455). In other words, you could likely patch `pyproject.toml` to remove that minimal requirement and build your own custom copy. But that would require forking the conda-forge feedstock to TileDB-Inc and then configuring it to build and upload to the `tiledb` channel